schemas/mw-page-delete.yaml

```diff
@@ -0,0 +1,53 @@
+title: MediaWiki Page Delete
```

Comment:
More info on this topic is available here: https://en.wikipedia.org/wiki/Wikipedia:Viewing_deleted_content